README.md

```diff
@@ -306,7 +306,10 @@ See the httpx documentation for information about the [`proxies`](https://www.py
 
 ### Managing HTTP resources
 
-By default we will close the underlying HTTP connections whenever the client is [garbage collected](https://docs.python.org/3/reference/datamodel.html#object.__del__) is called but you can also manually close the client using the `.close()` method if desired, or with a context manager that closes when exiting.
+By default, we will close the underlying HTTP connections whenever the client is 
```

Comment:
We fixed the typo in this sentence (`is called`), but we would like to keep the original text other than that, since we use passive tense elsewhere. The updated text will be in the next version